Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -69,6 +69,7 @@
 --rules redundantInit
 --rules redundantVoidReturnType
 --rules redundantOptionalBinding
+--rules redundantInternal
```

Comment:
Before merging I'll have to update this repo to point to a SwftFormat version that includes the new rule implementation